src/overlays/actors/ovl_En_Eiyer/z_en_eiyer.h

```diff
@@ -6,9 +6,18 @@
 
 struct EnEiyer;
 
+typedef void (*EnEiyerActionFunc)(struct EnEiyer *, GlobalContext*);
+
 typedef struct EnEiyer {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x188];
+    /* 0x014C */ SkelAnime skelanime;
+    /* 0x0190 */ EnEiyerActionFunc actionFunc;
+    /* 0x0194 */ s16 actionTimer;
+    /* 0x0196 */ s16 targetYaw;
+    /* 0x0198 */ Vec3s jointTable[19];
+    /* 0x020A */ Vec3s morphTable[19];
+    /* 0x027C */ Vec3f basePos;
+    /* 0x0288 */ ColliderCylinder colCyl;
```

Comment:
```suggestion
    /* 0x0288 */ ColliderCylinder collider;
```
We usually call them collider unless there is more than one collider type.